metacat-thrift/src/main/java/com/netflix/metacat/thrift/CatalogThriftHiveMetastore.java

```diff
@@ -1704,14 +1709,18 @@ private TException unimplemented(final String methodName, final Object[] args) t
             log.error(e.getMessage(), e);
             throw e;
         } catch (Exception e) {
-            CounterWrapper.incrementCounter("dse.metacat.thrift.counter.failure." + methodName);
+            registry.counter(registry.createId(LogConstants.CounterThrift.name() + "." + methodName)
+                .withTags(LogConstants.getStatusFailureMap())).increment();
             final String message = String.format("%s -- %s failed", e.getMessage(), methodName);
             log.error(message, e);
             final MetaException me = new MetaException(message);
             me.initCause(e);
             throw me;
         } finally {
-            log.info("+++ Thrift({}): Time taken to complete {} is {} ms", catalogName, methodName, timer.stop());
+            final long duration = registry.clock().monotonicTime() - start;
+             this.registry.timer(LogConstants.TimerThriftRequest.name()
+                  + "." + methodName).record(duration, TimeUnit.MILLISECONDS);
+            log.info("+++ Thrift({}): Time taken to complete {} is {} ms", catalogName, methodName, duration);
```

Comment:
Do we really want all this to be info level? Seems to me they might be better as debug? Not sure how @ajoymajumdar has this overall